src/modules/orders/use-cases/parse-sample-data/parse-sample-data.controller.ts

```diff
@@ -60,6 +64,17 @@ const parseSampleDataController = loggedController(
                 CreateOrderUseCaseFactory(type);
             const order: Order<SampleEntry<SampleEntryTuple>[]> =
                 await createOrder.execute(submissionFormInput);
+            const uploadedExcelVersion =
```

Comment:
Maybe extract this functionality into a use-case?

Reply:
done!